src/functions/on-call/create-on-call-data-from-json/handler.ts

```diff
@@ -0,0 +1,77 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
+import { middyfy } from "@libs/lambda";
+import type { OnCallImportEntry, OnCallImportError } from "src/database/models/oncall";
+
+/**
+ * Lambda method for importing on-call data from JSON
+ *
+ * Expects body to be an array of objects with "Week" (number) and "Person" (string) properties
+ * and a query parameter "year" (number)
+ */
+export const onCallImportFromJsonHandler = async (event) => {
```

Comment:
Looks good, only change I would suggest is we make sure the requesting user (via token) has admin credentials. We already have this in the utils, I think Juuso refactored it today so might be worth checking usage with him.